app/services/idv/steps/ipp/ssn_step.rb

```diff
@@ -20,6 +23,15 @@ def extra_view_variables
         def form_submit
           Idv::SsnFormatForm.new(current_user).submit(permit(:ssn))
         end
+
+        def updating_ssn
+          flow_session.dig(:pii_from_user, :ssn).present?
+        end
+
+        def threatmetrix_session_id
```

Comment:
maybe we call this `generate_threatmetrix_session_id`? since the value will long term live in `flow_session`

```suggestion
        def generate_threatmetrix_session_id
```